src/routes/AccountRoute/AccountRouteStyled.js

```diff
@@ -51,40 +54,49 @@ box-sizing: border-box;
     margin: 0;
     padding: 0;
     width: 100%;
+    max-width: 450px;
 }
 `;
 
 export const AvatarBlock = styled.div`
 text-align: center;
 grid-area: avatar;
+display: flex;
+flex-direction: column;
+align-items: center;
+gap: 8px;
 
 @media (max-width: 767px) {
-    display: flex;
-    margin: 12px 12px 24px 12px;
+    flex-direction: row;
+    margin: 12px 12px 24px 12px; 
 }
 `
 
-export const Avatar = styled.div`
+export const Avatar = styled.img`
 width: 256px;
 height: 256px;
 border-radius: 8px;
-background-color: var(--color-grey-12);
+background-color: var(--color-grey-3);
+object-fit: cover;
 
 @media (max-width: 767px) {
     width: 64px;
     height: 64px;
 }
 `;
 
-export const ChangeAvatar = styled(Button)`
-padding-top: 8px;
-color: var(--color-brand-blue);
-width: 120px;
 
-@media (max-width: 767px) {
-    padding: 8px 16px;
-}
-`
+export const AvatarVisuallyHiddenInput = styled('input')({
```

Comment:
Probably here has to be `... = styled.input'...' `?

Reply:
removed because was not needed.